drms/utils.py

```diff
@@ -72,10 +72,11 @@ def to_datetime(tstr, force=False):
     result : pandas.Series or pandas.Timestamp
         Pandas series or a single Timestamp object.
     """
-    s = pd.Series(tstr)
+    s = pd.Series(tstr).astype(str)
     if force or s.str.endswith('_TAI').any():
         s = s.str.replace('_TAI', '')
         s = s.str.replace('_', ' ')
         s = s.str.replace('.', '-', n=2)
     res = _pd_to_datetime_coerce(s)
-    return res.iloc[0] if isinstance(tstr, six.string_types) else res
+    valid_types = (six.string_types, six.integer_types, float)
```

Comment:
This was exactly what I was looking for! :D
Actually, I tried using `(len(res) == 1)` but it would give a scalar result if the list had one element only. I had no clue how to check whether it was a "scalar" quantity. I tried searching for functions that would directly return whether it is (int or float or str), but I didn't get any.
This is the best way to do this, I think. :)